cygnus-arcgis/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIArcgisFeatureTableSink.java

```diff
@@ -0,0 +1,789 @@
+                    LOGGER.debug("[NGSIArcgisAggregator-contextAttrToFeature] Processing Attribute: "
+                            + contextAttribute.toString());
+                    String attrName = contextAttribute.getName();
+                    String attrType = contextAttribute.getType();
+                    // get attribute value as is, quoted or not
+                    JsonElement attrValue = contextAttribute.getValue();
+
+                    // TODO crop metadata values
+                    LOGGER.debug("[NGSIArcgisAggregator] aggregate - Atribute (name --> " + attrName + ", type --> "
+                            + attrType + ", value --> " + attrValue + ", metadata= " + "ignored " + ")");
+
+                    // Get Feature attributes
+                    LOGGER.debug("Proccessing attribute value:  " + attrName + " --> " + attrValue);
+
+                    jsonElementToFeatureAttr(attrName, attrType, attrValue, feature);
+
+                } // for
+            } // if else
+        }
+    } // NGSIArcgisAggregator
+
+    /**
+     * Extracts Feature attributes from JSON.
+     * 
+     * @param attrName
+     * @param attrType
+     * @param attrValue
+     * @param feature
+     */
+    protected void jsonElementToFeatureAttr(String attrName, String attrType, JsonElement attrValue, Feature feature) {
+
+        // Get Feature attributes
+        LOGGER.debug("Proccessing attribute value:  " + attrName + " --> " + attrValue);
+
+        attrType = attrType.trim().toLowerCase();
+
+        switch (attrType) {
+
+            case "geo:json":
+                try {
+                    JsonObject location = attrValue.getAsJsonObject();
+                    if (location.get("type").getAsString().equals("Point")) {
+                        JsonArray coordinates = location.get("coordinates").getAsJsonArray();
+                        double latitude = coordinates.get(0).getAsDouble();
+                        double longitude = coordinates.get(1).getAsDouble();
+
+                        Point point = new Point(latitude, longitude);
+                        feature.setGeometry(point);
+
+                    } else {
+                        LOGGER.warn("Invalid geo:json type, only points allowed: " + location.toString());
+                    }
+                } catch (Exception e) {
+                    LOGGER.error("Invalid geo:json format, (sikipped): " + attrValue.toString() + " - Error: "
+                            + e.getMessage());
+                }
+                break;
+            case "boolean":
+                if (attrValue.isJsonPrimitive() && attrValue.getAsJsonPrimitive().isBoolean()) {
+                    feature.addAttribute(attrName, attrValue.getAsBoolean());
+                } else {
+                    String strValue = attrValue.getAsString().toLowerCase().trim();
+                    boolean result = "true".equals(strValue) || "1".equals(strValue);
```

Comment:
That "1" String is so mysterious. It would be better to add it as a constant on the class

Reply:
Fixed 
https://github.com/Stdri/fiware-cygnus/blob/df4b24bbcced902efbd41a66da84356ed8ef6435/cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIArcgisFeatureTableSink.java#L602